laske_export/exporter.py

```diff
@@ -242,3 +246,81 @@ def export_invoices(self, invoices):
         laske_export_log_entry.save()
 
         return laske_export_log_entry
+
+    def export_land_use_agreement_invoices(self, invoices):
+        """
+        :type invoices: list of Invoice | Invoice
+        :rtype: LaskeExportLog
+        """
+        if isinstance(invoices, Invoice):
+            invoices = [invoices]
+
+        now = timezone.now()
+        laske_export_log_entry = LaskeExportLog.objects.create(started_at=now)
+
+        sales_orders = []
+        log_invoices = []
+        invoice_count = 0
+
+        self.write_to_output(
+            "Going through {} land use agreement invoices".format(len(invoices))
+        )
+
+        for invoice in invoices:
+            self.write_to_output(" Land use agreement invoice id {}".format(invoice.id))
+
+            sales_order = SalesOrder()
+            set_constant_laske_values(sales_order)
+
+            adapter = LandUseAgreementInvoiceSalesOrderAdapter(
+                invoice=invoice, sales_order=sales_order,
+            )
+            adapter.set_values()
+
+            sales_orders.append(sales_order)
+            log_invoices.append(invoice)
+
+            invoice_count += 1
+
+            self.write_to_output(
+                " Added invoice id {} as invoice number {}".format(
+                    invoice.id, invoice.number
+                )
+            )
+
+        if invoice_count > 0:
+            self.write_to_output(
+                "Added {} invoices to the export".format(invoice_count)
+            )
+
+            sales_order_container = SalesOrderContainer()
+            sales_order_container.sales_orders = sales_orders
+
+            laske_export_log_entry.land_use_agreement_invoices.set(log_invoices)
+
+            export_filename = "MTIL_IN_{}_{:08}.xml".format(
+                settings.LASKE_VALUES["sender_id"], laske_export_log_entry.id
+            )
+
+            self.write_to_output("Export filename: {}".format(export_filename))
+
+            xml_string = sales_order_container.to_xml_string()
+
+            self.save_to_file(xml_string, export_filename)
+
+            self.write_to_output("Sending...")
+
+            # self.send(export_filename)
```

Comment:
Should this be commented?

Reply:
hmm.. nope :) It was just while debugging. Good!